galley/pkg/config/processor/transforms/transformer/provider.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package transforms contains basic processing building blocks that can be incorporated into bigger/self-contained
```

Comment:
It might be better to move this package to `processing/transformer`.

processing package contains generic building blocks, and processor package contains Galley/Istio specific ones. This is a generic building block.

Reply:
Done